app/src/main/java/org/greenstand/android/TreeTracker/managers/UserLocationManager.kt

```diff
@@ -55,7 +54,12 @@ class UserLocationManager(private val locationManager: LocationManager,
     fun startLocationUpdates(): Boolean {
         return if (hasLocationPermissions()) {
             if (!isUpdating) {
-                locationManager.requestLocationUpdates(LocationManager.GPS_PROVIDER, 0, 0f, locationUpdateListener)
+                locationManager.requestLocationUpdates(
+                    LocationManager.GPS_PROVIDER,
+                    0,
```

Comment:
We need to tweak the values. Currently I observe location update every second. This will have some battery consumption. Can we update this to every 5 seconds?

Also, we have this location data stream all the time when any fragments  of Treetracker app is in use. Do we really need it?

Reply:
I agree, this can be an issue for battery. If it's every 5 second though, the tree capture data may not be accurate. 

I know we're working on a design that doesn't have the map on the home screen, and that will help a lot in reducing GPS usage once we redesign the app UI.